From ebf36266d52a5936a5440174ee08227c74d096e4 Mon Sep 17 00:00:00 2001
From: Michael Schmitz <schmitz@biophys.uni-duesseldorf.de>
Date: Sun, 19 Apr 2009 03:24:40 +0200
Subject: [PATCH] m68k: Atari SCC verbosity fix

Fix verbosity level of SCC driver (leftover 'info' message) as reported by
Patrice Mandin.

Signed-off-by: Michael Schmitz <schmitz@debian.org>
Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/char/atari_scc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/char/atari_scc.c b/drivers/char/atari_scc.c
index 2bbd44c..7d7abe5 100644
--- a/drivers/char/atari_scc.c
+++ b/drivers/char/atari_scc.c
@@ -1351,7 +1351,7 @@ static int scc_open(struct tty_struct *tty, struct file *filp)
 	pr_debug(KERN_WARNING "SCC: enable rx ints ...\n");
 	scc_enable_rx_interrupts(port);
 	pr_debug(KERN_WARNING "SCC: enable rx ints done!\n");
-	pr_info("SCC: open port done!\n");
+	pr_debug("SCC: open port done!\n");
 	return 0;
 }
 
-- 
1.5.6.5

